app/models/activity.rb

```diff
@@ -66,6 +66,7 @@ class Activity < ApplicationRecord
   scope :funds, -> { where(level: :fund) }
   scope :programmes, -> { where(level: :programme) }
   scope :publishable_to_iati, -> { where(form_state: :complete, publish_to_iati: true) }
+  scope :project_activities, -> { where(level: :project).or(where(level: :third_party_project)) }
```

Comment:
There are scopes call `funds` and `programmes` already, and this seems to be breaking from the pattern, although I accept that this includes `projects` *AND* `third_party_projects` together.